libafl/src/schedulers/accounting.rs

```diff
@@ -282,7 +277,7 @@ where
     }
 
     /// Creates a new [`CoverageAccountingScheduler`] that wraps a `base` [`Scheduler`]
-    /// and has a default probability to skip non-faved [`Testcase`]s of [`DEFAULT_SKIP_NON_FAVORED_PROB`].
+    /// and has a default probability to skip non-faved Testcases of [`DEFAULT_SKIP_NON_FAVORED_PROB`].
```

Comment:
Why remove the link?

Reply:
`Testcase` is not used in Rust code (but only in this comment) so I deleted the `use` declaractions.

Reply:
Ok, fair